pkg/controllers/user/pipeline/controller/pipelineexecution/deploy.go

```diff
@@ -128,6 +133,27 @@ func (l *Lifecycle) deploy(projectName string) error {
 	return l.reconcileRb(projectName)
 }
 
+func (l *Lifecycle) waitResourceQuotaInitCondition(namespace string) error {
+	tries := 0
+	for tries <= 3 {
+		ns, err := l.namespaces.Get(namespace, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		set, err := namespaceutil.IsNamespaceConditionSet(ns, resourcequota.ResourceQuotaInitCondition, true)
+		if err != nil {
+			return err
+		}
+		if !set {
```

Comment:
Nit: Use If set instead of if !set, and you won’t need continue in your code.

Reply:
Updated